api.py

```diff
@@ -1,11 +1,31 @@
-
-from flask import Flask
+from flask import Flask, request
+from banco import Banco
 
 app = Flask(__name__)
 
 @app.route('/')
-def hello():
-   return "Bem vindo"
+def ola():
+
+    return "bem vindo"
+
+@app.route("/pessoa", methods=['GET','POST'])
```

Comment:
```suggestion
@app.route("/pessoa", methods=['POST'])
```
Caso a ideia seja manter o GET, precisaria fazer uma estratégia dentro do método:
https://flask.palletsprojects.com/en/1.1.x/quickstart/#http-methods